modules/microshift-install-rhel-image-mode-cni.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-install-rhel-image-mode.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-rhel-image-mode-run-image-cni_{context}"]
+= Checking the {microshift-short} bootc image for the CNI
+
+The {microshift-short} OVN-Kubernetes (OVN-K) Container Network Interface (CNI) requires the Open vSwitch service. The `openvswitch` kernel module must be available in the `bootc` image.
+//can we include this in the containerfile?
+
+.Prerequisites
+
+* A {op-system-base-full} 9.4 host with an active Red Hat subscription for building {microshift-short} bootc images and running containers.
+* You are logged into the {op-system} 9.4 host using the user credentials that have SUDO permissions configured.
+
+.Procedure
+
+. Check for the `openvswitch` module presence by running the following command:
++
+[source,terminal]
+----
+$ find /lib/modules/$(uname -r) -name "openvswitch*"
+----
++
+.Example output
+[source,terminal]
+----
+/lib/modules/6.9.9-200.fc40.x86_64/kernel/net/openvswitch
+/lib/modules/6.9.9-200.fc40.x86_64/kernel/net/openvswitch/openvswitch.ko.xz
+----
+
+. Set the `IMAGE_NAME` environment variable by running the following command:
++
+[source,termimal]
+----
+$ IMAGE_NAME=microshift-4.16-bootc
+----
+
+. Check the version of the kernel-core `package` used in the bootc image by running the following command:
++
+[source,terminal]
+----
+$ sudo podman inspect "${IMAGE_NAME}" | grep kernel-core
+        "created_by": "kernel-core-5.14.0-427.26.1.el9_4.x86_64"
+----
++
+[NOTE]
+====
+When a `bootc` image is started as a container, it uses the host kernel, which is not necessarily the same one used for building the image. If there is a kernel version mismatch with the modules present in the `/lib/modules` directory, the `openvswitch` module cannot be loaded in the container. To work around this problem, pre-load the `openvswitch` module before starting the container if you see different kernel versions like the ones shown in this example.
```

Comment:
This note needs to be before the commands to explain first what we're trying to achieve.